server/src/main/java/org/opensearch/crypto/CryptoHandlerRegistry.java

```diff
@@ -27,19 +26,19 @@
 
 /**
  * During node bootstrap, installed key provider extensions responsible for generating data keys are loaded.
- * Crypto factories against the respective extensions are cached. A crypto factory is used to register crypto
- * manager against an {@link org.opensearch.common.blobstore.EncryptedBlobStore}
+ * Crypto factories against the respective KP plugins are cached. A crypto factory is used to register crypto
+ * handler against an {@link org.opensearch.common.blobstore.EncryptedBlobStore}
  */
-public class CryptoManagerRegistry {
-    private static final Logger logger = LogManager.getLogger(CryptoManagerRegistry.class);
+public class CryptoHandlerRegistry {
+    private static final Logger logger = LogManager.getLogger(CryptoHandlerRegistry.class);
     // Package private for tests
     SetOnce<Map<String, CryptoKeyProviderPlugin>> registry = new SetOnce<>();
 
     // Package private for tests
-    SetOnce<CryptoManagerFactory> cryptoManagerFactory = new SetOnce<CryptoManagerFactory>();
-    private final Map<CryptoMetadata, CryptoManager> registeredCryptoManagers = new HashMap<>();
+    SetOnce<CryptoPlugin> cryptoHandlerPlugin = new SetOnce<>();
```

Comment:
replace `CryptoPlugin` with `CryptoPlugin<?,?> to avoid `Raw use of parameterized class 'CryptoPlugin' `